features/cli/cli_detach.feature

```diff
@@ -0,0 +1,171 @@
+@uses.config.contract_token
```

Comment:
If I can make a suggestion, let's get rid of this tag?

I wouldn't say we are running tests without setting a contract token anyway, now that it is way easier using the `protest.yaml` file.
And if we do, I'd say the error is pretty straightforward in telling us to add a token.
So this is not being used for filtering, and enforcing is useless. Besides that, right now it's not standardized through the codebase - here and there you will find a test that needs a token but lacks the tag.